### DISCUSSION TITLE
expense-shell frontend.sh

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1625] [Q&A] [answered]
![Screenshot 2023-11-21 164209](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/43a5471b-f6d7-4f1f-8f42-eee6cbfa126f)
last two lines error code is there

**Comment by sreedharm07** [comment]
@palsachennaiah 
remove this local host there in expence.conf file
instead of local host we should paste there our ip or dns record
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3d00e31f-c331-4a05-b8b8-648259594b38)

**Reply by palsachennaiah** [reply]
I changed problem_ not solved and below
![Screenshot 2023-11-21 163558](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/a3798c05-5624-46b5-a943-b5f2dd8bc6b3)

expense.conf 3 + -- (red colour) frontend.sh 7 ------- red colour what those indication

**Reply by sreedharm07** [reply]
that indicates 
-- means removed somrthing in that file and ++ means added something in that file

**Reply by sreedharm07** [reply, marked as answer]
why ip also you given  here.
give ip or dns record  any one you should paste there not both
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ba9131bf-5e8d-482a-80fb-e62e2ab04e36)

**Reply by palsachennaiah** [reply]
Ok i got it thank you
